bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Path.java

```diff
@@ -436,6 +439,7 @@ void cubicToInPixels(float cx1, float cy1, float cx2, float cy2, float x, float
 
 @Override
 void destroy() {
+	handleMap.values().forEach(handle -> Gdip.GraphicsPath_delete(handle));
 	Gdip.GraphicsPath_delete(handle);
```

Comment:
That handle is already deleted via the handleMap, right?

Reply:
Dont remember lemme check

Reply:
yes

Reply:
removed 443